src/karmabot/db/database.py

```diff
@@ -0,0 +1,78 @@
+import logging
+import sys
+from contextlib import contextmanager
+
+import sqlalchemy as sa
+import sqlalchemy.orm as orm
+from sqlalchemy.exc import OperationalError
+
+from karmabot import settings
+from karmabot.db.modelbase import SqlAlchemyBase
+from karmabot.exceptions import NotInitializedException
+
+
+class Database:
```

Comment:
So we got rid of the globals right? e.g. `__factory` - big WIN

Reply:
🚀